multi-agent/tests/unit/sharing/test_share_cloner_builtin_handling.py

```
@@ -0,0 +1,129 @@
+        name=f"custom-{rid}",
+        cfg_dict=deps_cfg or {},
+    )
+
+
+class TestComputeClosureBuiltinHandling:
+    """Tests for built-in resource handling in _compute_closure."""
+
+    def test_public_builtin_skipped_no_error(self) -> None:
+        """A PUBLIC built-in dependency is silently skipped (not cloned)."""
+        public_builtin = _make_builtin_resource("builtin-1")
+        cloner, resources_service, element_registry, _builtin_svc = _build_cloner(
```

Ruff (0.16.1)

[warning] Unpacked variable `element_registry` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)